go.mod

```diff
@@ -1,17 +1,58 @@
 module github.com/k14s/kbld
 
-go 1.16
+go 1.17
 
 require (
 	github.com/cppforlife/cobrautil v0.0.0-20200514214827-bb86e6965d72
 	github.com/cppforlife/go-cli-ui v0.0.0-20200716203538-1e47f820817f
 	github.com/google/go-containerregistry v0.7.0
 	github.com/hashicorp/go-version v1.3.0
-	github.com/kisielk/errcheck v1.6.0
+	github.com/kisielk/errcheck v1.5.0
```

Comment:
Any idea why this got downgraded?

Reply:
Looks like the `v1.6.0` tag is associated to a release named `Add SelectorName` - https://github.com/kisielk/errcheck/releases

My guess is that there is a mixup with the release name not matching a version string.

Reply:
I can update the PR to reference the `v1.6.0`tag